src/cmd-buildextend-metal

```diff
@@ -227,7 +232,19 @@ runvm_with_cache_snapshot "$snapshot" -- /usr/lib/coreos-assembler/runvm-osbuild
 
 if [[ "${image_type}" == "qemu-secex" ]]; then
     if [ ! -f "${genprotimgvm}" ]; then
-        fatal "No genprotimgvm provided at ${genprotimgvm}"
+        if [ ! -f "${hostkey}" ]; then
+            fatal "No hostkey and no genprotimgvm provided"
+        fi
+        echo "Generating genprotimgvm locally"
+        ignition=$(mktemp -p "${tmp_builddir}")
+        butane -p -d "$(dirname "${hostkey}")" /usr/lib/coreos-assembler/secex-genprotimgvm-scripts/genprotimg.bu -o "${ignition}"
+
+        genprotimgvm=/srv/fake-secure-vm.qcow2
+        cp "/srv/builds/latest/${basearch}/${name}-${build}-qemu.${basearch}.${image_format}" "${genprotimgvm}"
+        chmod +w "${genprotimgvm}"
+        genvm_args=("-drive" "if=none,id=hda,file=${genprotimgvm},auto-read-only=off,cache=unsafe" \
+                    "-device" "virtio-blk,drive=hda,bootindex=1")
+        kola qemuexec -i "${ignition}" -- "${genvm_args[@]}"
```

Comment:
Feels like we should skip this if it already exists. (I guess the host key might've changed. Not sure it's worth worrying about for now. We could in the future add a checksum of the key in the filename.)